src/cs/Ssh.Tcp/Services/RemotePortForwarder.cs

```diff
@@ -70,17 +72,29 @@ public class RemotePortForwarder : RemotePortConnector
 		// when the PortForwardingService is diposed.
 #pragma warning disable CA2000 // Dispose objects before losing scope
 		var tcpClient = new TcpClient();
+
+		// The event handler may return a transformed channel stream.
+		var forwardedStream = await pfs.OnForwardedPortConnectingAsync(
```

Comment:
Should we catch exceptions in the `ForwardedPortConnecting` handler and reject the request too?

Reply:
Updated.